### ISSUE TITLE
Adding silent property on api level for view methods to prevent infinite event recursion.

### ISSUE THREAD

**Issue by WebKieth** [openlayers/openlayers#14506]
PS, sorry for my english, I'm ru =)
**Is your feature request related to a problem? Please describe.**
I develop the project of interface, which can syncronize multiple instances of maps. This is difficult technology application case, when maps using by medical analytics. The point is to synchronize the movement through the layers of several biological cell sections.
In first view we can check primary movement event from `view.on('change:<property>')`. But if we do it and try to set new property by view methods, we run into a problem of recursive event calls. Event calls set method, which calls event it goes on endlessly.
As workaround I create my own external event bus module and dispatch it from custom interactions, so as result I have got a heavy codebase, which I want to simplify.

**Describe the solution you'd like**
I edit View class and take out silent property on api level. Please, write what you think about this idea. If you like it, I can [pull my commit](https://github.com/openlayers/openlayers/compare/main...WebKieth:openlayers:view_silent_set) to your original project.

**Comment by M393** [comment]
You can use the same view object for multiple maps: 
https://openlayers.org/en/latest/examples/side-by-side.html

**Comment by WebKieth** [comment]
@M393 no, I can't. Feature of syncronization can be enabled and disabled with different layers with different scales.

**Comment by mike-000** [comment]
If you maintain a global sync flag you can prevent further actions when a sync is already in progress.  Use the completion callback for a 0 duration animation to clear the flag when the first action is complete.  e.g. make a change to the center, zoom or rotation of one of the maps in https://codesandbox.io/s/sync-changes-4u5uol?file=/main.js then sync the maps, they will move together but maintain the difference set before syncing.
